docs/playground/DESIGN.md

```diff
@@ -0,0 +1,225 @@
+    only the application but also OS/OSD. SSL certificates must be made and
+    used.
+- **Easily Accessible:** the application should not require the user to 
+    download, sign-up, or any other extraneous step to play around with 
+    OS/OSD's features.
+- **Scaleable:** because this application showcases the capabilities of 
+    OS/OSD, this application will be a first impression for many customers.
+    Thus, the application should service them in a fast, secure, and reliable
+    manner.
+
+
+## Proposal
+
+To address this need, a demo website we call OpenSearch Playground was 
+proposed that would allow users to visit and start experimenting with OS/OSD. 
+To meet our goal of an accessible, scaleable application, the application 
+will leverage the speedy, open-source container orchestrator Kubernetes and 
+Amazon Web Service's Elastic Kubernetes Service to enable fast, reliable 
+delivery of OS/OSD to our customers.
+
+
+# Who is Affected
+
+The need for OpenSearch Playground site in particular, targets all of 
+OpenSearch's customers, both existing and prospective. They will be the users 
+who will learn and be curious about OS and OSD. Because this application will 
+be for many of them a first hands-on experience, it is vital that OpenSearch Playground delivers the optimal experience of OS and OSD. 
+
+OpenSearch Playground not only serves customers who will want to potentially utilize OS and OSD but also the contributors, who want a one-stop application 
+to familiarize themselves with the site in order to start contributing or make
+improvements to their existing work. Thus, the site serves a multifaceted 
+purpose and targets a diverse audience.
+
+
+# Design
+
+Leveraging the power and scale of Amazon Web Service's **(AWS)** Elastic 
+Kubernetes Service **(EKS)** as well as Route 53, OpenSearch Playground allows 
+any prospective users to experiment with OS and OSD. OpenSearch Playground will 
+be open to all (requiring no sign-ups) as a way to showcase the robust 
+capabilities of the services offered by OS/OSD via a secure https connection.
+
+![diagram_white](https://user-images.githubusercontent.com/73027756/173409473-3cc075f3-8ce6-4eb4-95e4-df6f13eb3899.png)
+
+Creating OpenSearch Playground first requires the modification of the OS and OSD bundle configurations . OpenSearch and OpenSearch Dashboards initially require certain permissions to access features, like viewing the dashboards and interacting with sample data. Thus, the images were configured so that anonymous users could view only the features via the enabling of settings in the `config.yml`, `roles.yml`, `roles_mapping.yml`, and `opensearch_dashboards.yml`. 
+
+Sample change for `config.yml`:
+```
+config:
+  dynamic:
+    http:
+      anonymous_auth_enabled: true
+```
+
+Sample change for `roles.yml`:
+```
+opendistro_security_anonymous:
+  reserved: true
+  cluster_permissions:
+    - 'cluster_monitor'
```

Comment:
what's kind of permission here for cluster_monitor ?

Reply:
This permission was required so that the user in Dev Tools can make some queries like `GET _cat/indices`.

Reply:
https://github.com/opensearch-project/security/blob/d0e06aa3def41402c121963e74147e40d1246741/src/test/resources/legacy/securityconfig_v6/migration/action_groups.yml#L109, you are give all the permission under cluster_monitor. as I said the previous comments, this is design doc, not implement document. you still need to add helm's opensearch and opensearch dashboard to config folder and it will be used for deployment.

Reply:
So do I just delete the sample configurations for all my files? I understand that this document is for design and so that is why I just outlined some changes I have made.